Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM selenium/standalone-firefox
+
+USER root
+
+ARG FIREFOX_DOWNLOAD_URL=https://download.mozilla.org/?product=firefox-nightly-latest-ssl&lang=en-US&os=linux64 
+
+RUN apt-get -qqy update && apt-get install -qqy \ 
+    git \
+    python \
+    python-dev \
+    python-pip \ 
+    build-essential \
+    && pip install --upgrade pip \
+    && rm -rf /var/lib/apt/lists/* /var/cache/apt/*
+
+RUN pip install --upgrade pip && pip install tox 
+
+RUN wget --no-verbose -O /tmp/firefox.tar.bz2 $FIREFOX_DOWNLOAD_URL \
+  && rm -rf /opt/firefox \
+  && tar -C /opt -xjf /tmp/firefox.tar.bz2 \
+  && rm /tmp/firefox.tar.bz2 \
+  && mv /opt/firefox /opt/firefox-nightly \
+  && ln -fs /opt/firefox-nightly/firefox /usr/bin/firefox
+
+EXPOSE 5900
+EXPOSE 4444
+
+WORKDIR /tests
+COPY . /tests
+RUN pip install -r /tests/requirements/requirements.txt
+
+
+CMD TEST_ENV=${TEST_ENV} pytest --channel="nightly" --pref-set="base" tests/test_tracking_protection.py -s 
```

Comment:
notice I'm not using tox (and I don't think we should either).  also, we'll probably want to consider if we really want Jenkins to be able to execute specific tests or pref-sets or if we're just going to assume all should be run by our CI.